test/r401_test.js

```diff
@@ -0,0 +1,336 @@
+const cql = require('cql-execution');
+const cqlfhir = require('../src/index');
+const {expect} = require('chai');
+
+const conditionResource = require('./fixtures/r4/Condition_f201.json');
+const patientMyron = require('./fixtures/r4/Luna60_McCullough561_6662f0ca-b617-4e02-8f55-7275e9f49aa0.json');
+const patientShawnee = require('./fixtures/r4/Johnnie679_Hermiston71_2cd30bd6-3a87-4191-af90-6daa70f58f55.json');
+
+describe('#FHIRWrapper_R4', () => {
```

Comment:
Update this describe be different than the 4.0.0 tests. In the current state it's a bit confusing to know which test results are for which FHIR version.

Reply:
Good point- I updated both descriptions to avoid confusion.

Reply:
I also removed the unused test fixture. As for the changes to the other unit tests, they were importing the same files multiple times, so I refactored them to be more efficient. I've grown accustomed to the ES6 and TypeScript modules, so seeing dynamic requires has begun to look like an anti-pattern to me. But I could revert them if you prefer.

Reply:
I think that makes sense. But a possible downside is that if a test ends up being destructive to the data loaded it could affect other tests. I think that could have been why it was done in the `beforeEach`. I'll leave this to @cmoesel if he thinks this refactoring is ok.

Reply:
For what it's worth, they're already modified in the `before()` in a safe way:
`conditionResourceWithNoType = JSON.parse(JSON.stringify(conditionResource));`
`delete conditionResourceWithNoType.resourceType;`